### ISSUE TITLE
Wrong lang attribute

### ISSUE THREAD

**Issue by eddiemf** [szczepienia/szczepienia.github.io#62]
The lang attribute is currently defaulting to `'en'` when it should be `'pl'` to facilitate accessibility (auto translate). I'm not familiar with Jekyll so I can't really help, but should be a simple change to the default layout.

**Comment by szczepienia** [comment]
Thanks, corrected.
